### PR TITLE
Implement `nrn_pointing`.

### PR DESCRIPTION
**Pull request by 1uc**
It seem `nrn_pointing` is a special builtin function that requires the
pointer, not the dereferenced value of a pointer.

### CODE REVIEW

**Comment by 1uc** [comment]
Requires #1462.

File: src/codegen/codegen_cpp_visitor.cpp [hunk 1]
```diff
@@ -524,6 +529,18 @@ void CodegenCppVisitor::print_function_call(const FunctionCall& node) {
     printer->add_text(')');
 }
 
+void CodegenCppVisitor::print_nrn_pointing(const ast::FunctionCall& node) {
+    const auto& args = node.get_arguments();
+    if (args.size() != 1) {
+        throw std::runtime_error(
+            fmt::format("nrn_pointing excepts exactly one argument, got: {}", args.size()));
+    }
```
Comment (pramodk) [inline src/codegen/codegen_cpp_visitor.cpp:537]: just a nitpick: checks like this are typically done in [the semantic analysis visitor](https://github.com/BlueBrain/nmodl/blob/2f448528977bd8b60e552123fc7195bd35cecf2b/src/visitors/semantic_analysis_visitor.hpp#L39).

**Review by pramodk — APPROVED** [review]
LGTM

File: src/codegen/codegen_cpp_visitor.cpp (hunk 1 above)
Comment (1uc) [reply src/codegen/codegen_cpp_visitor.cpp]: ty, changed.

**Pull request merged**